base/utils/R/days_in_year.R

```diff
@@ -11,9 +11,9 @@
 #' days_in_year(2010)  # Not a leap year -- returns 365
 #' days_in_year(2012)  # Leap year -- returns 366
 #' days_in_year(2000:2008)  # Function is vectorized over years
-days_in_year <- function(year) {
+days_in_year <- function(year, leap_year = TRUE) {
```

Comment:
This argument should be documented. Add something like the following to the Roxygen block above this:

```r
#' @param leap_year Whether or not to enforce leap years. If `FALSE`, always return a 365 day year.
```

Reply:
Also, once you do this, you should re-run `make document` to make sure the documentation is updated.

Reply:
Woops. Thanks for catching that.